index.html

```diff
@@ -0,0 +1,186 @@
+        <div class="line"></div>
+    </div>
+    <section class="main-wrapper">
+    <section class="p-wrapper">
+        <div class="img">
+            <img src="assets/adobe-photoshop.png" alt="">
+         </div> 
+        <h3>Graphic Design</h3>
+        <p>here you will learn graphic designs</p>
+    </section>
+    <section class="p-wrapper">
+        <div class="img">
+            <img  src="assets/html (1).png" alt="graphic-design-image">
+        </div>
+        <h3>Front-End Design</h3>
+        <p>become a front-end developer</p>
+    </section>
+   <section class="p-wrapper">
+        <div class="img">
+            <img  src="assets/budget.png" alt="graphic-design-image">
+        </div>
+        <h3>Acounting</h3>
+        <p>learn math and apply to real world</p>
+    </section>
+    <section class="p-wrapper">
+        <div class="img">
+            <img  src="assets/goal.png" alt="graphic-design-image">
+        </div>
+        <h3>Project Planning</h3>
+        <p>manage real projects by learning</p>
+    </section>
+    <section class="p-wrapper">
+        <div class="img">
+            <img  src="assets/cyber-security.png" alt="cyber-security-image">
+        </div>
+        <h3>Cyber Security</h3>
+        <p>Secure your information from malicious</p>
+        </div>
+    </section>
+</section>
+ <a href="#" class="btn-href"><button type="button" class="bttn">Join Genius Academy now!</button></a>
+</main>
+
+<section class="featured-teachers desk-version" >
+    <div>
+        <h2>Featured Teacher</h2>
+        <div class="feature-line"></div>
+    </div>
+    <div class="container">
+    <section class="teacher-wrapper">
+        <div class="teacher-img">
+            <img src="assets/speaker-back.png" alt="">
+            <img src="assets/teacher-five-removebg-preview.png" alt="" class="overlay-one">
+            <img src="assets/speaker-back.png" alt="speaker-back-image">
+            <img src="assets/teacher-five-removebg-preview.png" alt="teacher-image" class="overlay-one">
+        </div>
+        <div class="right-content">
+            <h3>Ilhan Muse</h3>
+            <h5>Lecturer at Amoud University</h5>
+            <hr>
+            <p>She has been teachin since 2016, her major is IT</p>
+        </div>
+    </section>
+    <section class="teacher-wrapper ">
+        <div class="teacher-img">
+            <img src="assets/speaker-back.png" alt="">
+            <img src="assets/teacher-two-removebg-preview.png" alt="" class="overlay-one">
+        </div>
+        <div class="right-content">
+            <h3>Abdala Ali</h3>
+            <h5>Telesom Company</h5>
+            <hr>
+            <p>He is a head of the technical department at the Telesom Company </p>
+        </div>
+    </section>
+    <button type="button" class="see-more">MORE
+        <i class="fa-solid fa-chevron-down chevron"></i>
+    </button>
+</div>
+</section>
+
+<section class="desk-featured-speaker" id="featuredteacher">
```

Comment:
- [ ] [OPTIONAL] Good so far :clap: But I think you can improve and I would strongly like to recommend you to make your partners section follow the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including: Colors, Layout: composition and space between elements to make your website look wonderful :+1: